src/prototype_creep.js

```diff
@@ -42,6 +42,7 @@ Creep.prototype.moveToMy = function(target, range) {
     visualizer.showSearch(search);
   }
 
+  this.creepLog('moveToMy search:' + JSON.stringify(search));
```

Comment:
this.creepLog('moveToMy search:', JSON.stringify(search));
// use join on string (+) or array of strings? (,)

Reply:
:+1 thx